lib/common.sh

```diff
@@ -527,27 +527,14 @@ differs(){
   return ${RET_CODE}
 }
 
-# If a given container with tag doesn't exist locally, pull it.
-# Otherwise, do nothing.
-# Helps conserve number of API calls to DockerHub to avoid hitting rate limit.
-#
 # Inputs:
 # - Full name of a Docker/podman/crictl image including tag
 #
-pull_container_image_if_missing() {
+pull_container_image() {
```

Comment:
this should be called somewhere ?

Reply:
yes the file was left out from the commit.